src/tools/importer/README.md

```diff
@@ -1,33 +1,149 @@
 
-#### Importer
+## Importer
 
-`Importer` is a tool use to load `CSV` file into `Nebula`.
+`Importer` is a tool used to load a small amount of data from a CSV file into Nebula Graph. It is single-threaded. If you have a large set of data, generating SST files and ingesting these files may be a better way. For details, please refer to [Spark SST File Generator](../spark-sstfile-generator/README.md) and HDFS Importer <!--to be done-->
```

Comment:
How about condensing the first two sentences into one?
`Importer` is a single-threaded tool used to load small data set from a CSV file into Nebula Graph.
As for large data set, generating SST files and then ingesting them may be a better way.